libs/api/domains/umbodsmadur-skuldara/src/lib/models/costOfLivingCalculator.model.ts

```diff
@@ -0,0 +1,41 @@
+import { CacheField } from '@island.is/nest/graphql'
+import { Field, ObjectType, Int } from '@nestjs/graphql'
+
+@ObjectType('CostOfLivingCalculatorModel')
+export class CostOfLivingCalculatorItem {
+  @Field(() => String)
+  numberOf!: string
```

Comment:
~This variable could also be more descriptive, number of what?

~Some variables are also postfixed with `cost` and others are not. Just thinking of consistency.